src/org/labkey/test/components/ui/ontology/OntologyTreeSearch.java

```diff
@@ -159,7 +159,7 @@ WebElement resultContainer()
     public static class OntologyTreeSearchFinder extends WebDriverComponentFinder<OntologyTreeSearch, OntologyTreeSearchFinder>
     {
         private final Locator.XPathLocator _baseLocator = Locator.tagWithClass("div", "concept-search-container");
-        private String _title = null;
+        private final String _title = null;
```

Comment:
it's part of the template, we're not using it at the moment (but it might be useful if we want to use the annotation field's input name attribute to find this component in the future)
For now it's dead code, I'll pull it out